ui/src/api/relations/relations.js

```diff
@@ -0,0 +1,59 @@
+import { http } from '@api/base';
+import {
+  add as parentChildAdd,
+  remove as parentChildRemove,
+} from './parentChild';
+import { add as siblingsAdd, remove as siblingsRemove } from './siblings';
+import { add as sequenceAdd, remove as sequenceRemove } from './sequence';
```

Comment:
nit: in previous imports 	`import { add as addRelation, remove as removeRelation } from '@api/relations';` used the actions as prefix, `addRelation` while on this one are suffix, `siblingsAdd`, I am in favour of prefixing :) `addSiblings`, `removeSiblings`, `addSequence` and `removeSequence` wdyt?

Reply:
done